proctor-common/src/main/java/com/indeed/proctor/common/ProctorUtils.java

```diff
@@ -1417,4 +1434,21 @@ private static boolean evaluatePayloadValidator(
 
         return false;
     }
+
+    public static boolean containsUnitlessAllocation(
```

Comment:
Team, I have been brainstorming, I plan on making this take a TestDefinition and storing this variable in consumable test definition so it is only calculated once at load from matrix to consumable test definition. Tests which lots of allocations could otherwise be more inefficient at each determineBuckets/Groups.

Reply:
Completed